From 84eae85d72f4e5369e2d1e996062bfe1a7b6ad80 Mon Sep 17 00:00:00 2001
From: Jason Parker <jparker@digium.com>
Date: Thu, 28 Feb 2013 21:21:50 +0000
Subject: [PATCH] Don't undefine bzero()/bcopy().

This was causing build failures against external libraries that happened to use
them, unless silly hacks were added to the modules that used those headers.

Review: https://reviewboard.asterisk.org/r/2359/

Backporting to 11, as pjproject has changed.

Origin: http://svnview.digium.com/svn/asterisk?view=rev&rev=382292
---
 include/asterisk.h     |   11 -----------
 res/res_rtp_asterisk.c |    4 ----
 2 files changed, 15 deletions(-)

--- a/include/asterisk.h
+++ b/include/asterisk.h
@@ -213,17 +213,6 @@ struct ast_variable;
 struct ast_str;
 struct ast_sched_context;
 
-#ifdef bzero
-#undef bzero
-#endif
-
-#ifdef bcopy
-#undef bcopy
-#endif
-
-#define bzero  0x__dont_use_bzero__use_memset_instead""
-#define bcopy  0x__dont_use_bcopy__use_memmove_instead()
-
 /* Some handy macros for turning a preprocessor token into (effectively) a quoted string */
 #define __stringify_1(x)	#x
 #define __stringify(x)		__stringify_1(x)
--- a/res/res_rtp_asterisk.c
+++ b/res/res_rtp_asterisk.c
@@ -47,10 +47,6 @@ ASTERISK_FILE_VERSION(__FILE__, "$Revisi
 #include <openssl/bio.h>
 #endif
 
-/* Asterisk discourages the use of bzero in favor of memset, in fact if you try to use bzero it will tell you to use memset. As a result bzero has to be undefined
- * here since it is used internally by pjlib. The only other option would be to modify pjlib... which won't happen. */
-#undef bzero
-#define bzero bzero
 #include "pjlib.h"
 #include "pjlib-util.h"
 #include "pjnath.h"
